security/step-ca/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/smallstep/certificates 0.18.1 v
+github.tarball_from archive
+name                step-ca
+revision            0
+
+description         A private certificate authority (X.509 & SSH) & ACME \
+                    server for secure automated certificate management, so \
+                    you can use TLS everywhere & SSO for SSH.
+
+long_description    step-ca is an online Certificate Authority (CA) for secure, \
+                    automated X.509 and SSH certificate management. It's the \
+                    server counterpart to step CLI. It is secured with TLS, and \
+                    it offers several configurable certificate provisioners, \
+                    flexible certificate templating, and pluggable database backends \
+                    to suit a wide variety of contexts and workflows. It employs sane \
+                    default algorithms and attributes, so you don't have to be a \
+                    security engineer to use it securely.
+
+categories          security
+installs_libs       no
+license             Apache-2
+maintainers         {gmail.com:wyuenho @wyuenho} \
+                    openmaintainer
+
+# Allow fetching deps at build time
+build.env-delete    GO111MODULE=off GOPROXY=off
+
+checksums           rmd160  45887526b3e2cb0dcf61d0324614cc9c6cd23bdc \
+                    sha256  f758012721876fee9f5c0f45933e2d09a26e3d4155d1bdbd99eac41d545739b0 \
+                    size    17887414
+
+# Mirrors already have a tarball named cli-${version}.  Since the tarball
+# is versioned, then we don't have to version our dist_subdir. We need only
+# create one with the project name just to differentiate from the tarball
+# already present.
+dist_subdir         ${name}
```

Comment:
Seems like you copied the Portfile for `step-cli`.  That being the case, you do not need to keep this `dist_subdir` line and its associated comment above.